src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/packages.config

```diff
@@ -13,11 +14,9 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.26.305102204" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.0" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime" version="1.8.2" targetFramework="net45" />
```

Comment:
if you remove this package from configuration you need to remove it from csproj as well. Then you will need to regenerate wxi file.